libs/pong/pong.js

```diff
@@ -45,7 +46,12 @@ export function createPongGetHandler(client, coder) {
           if (v === null || v?.[0] === null) {
             return [p, null];
           }
-          if ((p === "side") | (p === "hpMain") | (p === "hpFooter")) {
+          if (
+            (p === "side") |
+            (p === "hpMain") |
+            (p === "hpFooter") |
+            (p === "bottom")
```

Comment:
Nit: Why bitwise OR?

```suggestion
            (p === "side") ||
            (p === "hpMain") ||
            (p === "hpFooter") ||
            (p === "bottom")
```